### ISSUE TITLE
Ajouter une page Mention Légales - dans le backoffice pour la rédaction + dans le front (option dans le menu)

### ISSUE THREAD

**Issue by bouviermullerp** [data-players/Opaline#33]

**Comment by simonLouvet** [comment]
@alicetatangwidjaja est ce que ce besoin est confirmé? est ce que vous voulez avoir la main pour modifier les mentions légales. Si c'est le cas il faut ajouter une notion de page ans la backoffice et la mise en page des mentions légales sera sobre/simple.
Il est également possible de faire une page sur laquelle vous n'aurez pas la main que je créé mais il faudra me fournir le contenu/design a chaque fois que vous voulez la modifier

**Comment by alicetatangwidjaja** [comment]
@simonLouvet oui, ce besoin est confirmé. La page devrait être éditable par nous, comme la page de FAQ. Et la page devrait être référencé dans le menu, sous FAQ. Voici pour le contenu initial : 


**Propriétaire et éditeur du site** 
Collectif emploi est un réseau animé par l’association Tous Tes Possibles. Siège : 51 rue Stanislas, 54000 NANCY.
N° SIREN : 824972665
Tel. du gestionnaire du site : 06 36 12 28 74

**Realisation du site** 
DataPlayers 

**Conditions générales d'utilisation** 
L’utilisateur reconnaît avoir pris connaissance des présentes conditions générales d’utilisation et s’engage à les respecter.
L’équipe Collectif emploi se réserve le droit de modifier la présente notice à tout moment.
L’utilisateur s’engage donc à la consulter régulièrement.

**Contenu du site** 
Le site Collectif emploi a pour objet le partage des actions collectives, des ressources et solutions créées, ainsi que de favoriser l’échange entre les membres de la communauté. L’équipe d’animateurs communautaires s’efforce de fournir sur ce site des informations aussi précises que possible. Toutefois, elle ne pourra être tenue responsable des omissions, des inexactitudes et des carences dans la mise à jour, qu’elles soient de son fait ou du fait des tiers partenaires qui lui fournissent ces informations.

Toutes les informations indiquées sur ce site sont données à titre indicatif, et sont susceptibles d’évoluer. Par ailleurs, les renseignements figurant sur ce site ne sont pas exhaustifs. Ils sont donnés sous réserve de modifications ayant été apportées depuis leur mise en ligne.

**Confidentialité - données personnelles** 

Données relatives à l’utilisateur
Les données relatives aux paramètres de connexion de l’utilisateur, recueillies ou traitées à l’occasion de son utilisation de l’outil, sont destinées à la communauté Collectif emploi et à l’association Tous Tes Possiibles qui les accompagne, afin d’assurer le bon fonctionnement de l’outil web, d’améliorer les services offerts, d’adapter l’affichage du contenu, sauf opposition de sa part.

Droit d’accès et de rectification et droit d’opposition des utilisateurs identifiés
L’utilisateur dûment identifié auprès du réseau Collectif emploi bénéficie d’un droit d’accès aux informations le concernant et, le cas échéant, d’un droit de rectification, ainsi que d’un droit d’opposition à leur conservation ou à leur utilisation à des fins promotionnelles.
